### PR TITLE
endpoints for creating learners 

### PR DESCRIPTION
**Pull request by otimebhor**
Endpoint for creating learners
Endpoint for file upload to create learners

### CODE REVIEW

File: src/database/migrations/1719251695229-create-prospective-students.ts
Comment (Tes-program) [inline src/database/migrations/1719251695229-create-prospective-students.ts:1]: Why did you change the approach that was previously there before it was working well? And did you test it to see if it was working? I am referring to to the down method.

**Review by Tes-program — CHANGES_REQUESTED** [review]

File: src/database/migrations/1719251695229-create-prospective-students.ts
Comment (otimebhor) [reply src/database/migrations/1719251695229-create-prospective-students.ts:1]: There was a change in the user table that wasn't reflecting on my database.  so I have to revert the migrations them again. So the migration for this file wasn't working so I had to make those changes and it worked.

**Review by Tes-program — APPROVED** [review]
